vss-extension.json

```diff
@@ -134,7 +142,59 @@
                     }
                 }]
             }],
-            "helpMarkDown": "Is your project open source? Have a look at <a href=\"https://sonarcloud.io\">sonarcloud.io</a> to freely consume SonarQube as a service."
+            
+                "helpMarkDown": "Is your project open source? Have a look at <a href=\"https://sonarcloud.io\">sonarcloud.io</a> to freely consume SonarQube as a service."
+            }
+        },
+        {
+            "id": "343c5f6a-e4d5-4480-a764-506e1daa05df",
+            "description": "Service endpoint type for SonarCloud Connections",
+            "type": "ms.vss-endpoint.service-endpoint-type",
+            "targets": [
+                "ms.vss-endpoint.endpoint-types"
+            ],
+            "properties": {
+                "name": "sonarcloud",
+                "displayName": "SonarCloud",
+                "icon": "icons/SonarCloud_icon100px.png",
+                "url": {
+                    "displayName": "Server Url",
+                    "value": "https://sonarcloud.io",
+                    "isVisible": false
+                },
+                "dataSources": [
+                    {
+                      "name": "TestConnection",
+                      "endpointUrl": "{{endpoint.url}}/api/organizations/search_my_organizations",
+                      "resultSelector": "jsonpath:$.organizations[*]"
```

Comment:
I think you are using 4 spaces tabs normally. I don't know what editor you are using and how you format ts, js or json file but maybe we could use prettier (formatting can be done automatically on save in vscode), that's what we are using in SonarQube.

Reply:
+1 to use prettier. Do we have a shared configuration somewhere?

Reply:
Ok cool, the prettier configuration is in the `package.json`, on SonarQube we usually follow the prettier "trend" (nearly no custom rules). And if a rule change a lot from a version to another and we want it we just do a commit with the new formatting.
And for vscode, I use : 
```
  "editor.formatOnType": false,
  "editor.formatOnSave": true, 
  "javascript.validate.enable": false,
  "javascript.format.enable": false,
  "files.autoSave": "afterDelay",
  "files.autoSaveDelay": 10000,
```